src/blocks/block-column-inner/components/column.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Column wrapper component.
+ */
+
+/**
+ * WordPress dependencies.
+ */
+const { Component } = wp.element;
+
+/**
+ * External dependencies.
+ */
+import classnames from 'classnames';
+
+/**
+ * Create a Columns wrapper Component.
+ */
+export default class Column extends Component {
+
+	constructor( props ) {
+		super( ...arguments );
+	}
+
+	render() {
+
+		const {
+			attributes,
+			backgroundColor,
+			textColor,
```

Comment:
`backgroundColor` and `textColor` can be removed here since they're referenced via `attributes.*` below.